src/common/definitions.js

```diff
@@ -311,6 +311,21 @@ const educationProgram = {
   }
 };
 
+const currentlyActiveDuty = {
+  type: 'object',
+  properties: {
+    yes: {
+      type: 'boolean'
+    },
+    onTerminalLeave: {
+      type: 'boolean'
+    },
+    nonVaAssistance: {
```

Comment:
this is supposed to be
```
11B. FOR ACTIVE DUTY CLAIMANTS ONLY: Are you receiving or do you anticipate receiving
 any money (including but not limited to Federal Tuition Assistance) from the Armed Forces
 or Public Health Service for the course for which you have applied to the VA for education
 benefits? If you receive such benefits during any part of your training, check "YES."
```
it's in the same object because the question is for people on active duty only. also it was in the same object for the 1990 and i wanted to keep things organized the same way whenever possible.